pkg/ccl/streamingccl/streamingest/replication_stream_e2e_test.go

```diff
@@ -929,9 +929,10 @@ func TestTenantStreamingShowTenant(t *testing.T) {
 		jobId         int
 		maxReplTime   time.Time
 		protectedTime time.Time
+		cutoverTime   []byte // should be nil
```

Comment:
Can we add a test that once a cutover is initiated, this isn't nil

Reply:
I'll add that to the datadriven.

Reply:
sorry, we already test that in the datadriven test, right?

Reply:
oh I meant that in addition to checking its nil when there's no cutover if we can ensure that cutover actually shows up in the output of `SHOW TENANT` once we have issued a `COMPLETE REPLICATION`. I'll leave it up to you if it seems necessary.

Reply:
good point, added at the end of this test.